Sources/UIImageColorsObjc/UIImageColorsScaleQuality.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UIImageColorsScaleQuality.swift
+//  UIImageColors
+//
+//  Created by Felix Herrmann on 14.10.21.
+//
+
+import Foundation
+
+#if SWIFT_PACKAGE
+import UIImageColors
+#endif
+
+/// The quality the original image should be scaled to.
+@objc
+public enum UIImageColorsScaleQuality: Int {
+    
+    /// Scales the image to 50 pixel.
+    case low
+    
+    /// Scales the image to 100 pixel.
+    case medium
+    
+    /// Scales the image to 250 pixel.
+    case high
+    
+    /// Does not scale the image.
+    case full
```

Comment:
`highest` makes more sense here. It follows the category of the other cases and implies it is the upperbound

Reply:
hm but this is a special case where nothing is scaled 🤔